psakefile.ps1

```diff
@@ -0,0 +1,55 @@
+include "./psake-build-helpers.ps1"
+
+$tag = $(git tag -l --points-at HEAD)
```

Comment:
since you're using GitVersion now, you don't need any of this anymore.

Reply:
I talked to Jimmy a while back. He suggested releasing the nuget package when the master is tagged and when the master is not tagged all commits go to the private azure feed. Everytime I push a tag the version set is not just 3.0.0 or 3.1.0 etc but it is of the format 3.0.0-tags-v3-4.36.

Reply:
with GitVersion then, you need to use the `releases/3.0.0` style tagging to do what you want it to do.